### DISCUSSION TITLE
laravellivewiretable is not defined Alpine

### DISCUSSION THREAD

**Discussion by ltmstudio** [rappasoft/laravel-livewire-tables#2052] [Q&A]
I have a lot of errors in console
Uncaught ReferenceError: laravellivewiretable is not defined
Uncaught ReferenceError: setTableId is not defined
Uncaught ReferenceError: shouldBeDisplayed is not defined
My view
`<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Laravel</title>
    
    <link rel="preconnect" href="https://fonts.bunny.net">
    <link href="https://fonts.bunny.net/css?family=figtree:400,500,600&display=swap" rel="stylesheet" />
    @livewireStyles
    
    @vite(['resources/css/app.css', 'resources/js/app.js'])
</head>
<body class="antialiased">

    <div x-data="{ currentlyReorderingStatus: false }">
        @livewire('users-table')
    </div>
        <script src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js" defer></script>
    @livewireScripts
</body>

</html>
`
Component
`<?php

namespace App\Livewire;

use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;
use App\Models\User;

class UsersTable extends DataTableComponent
{
    protected $model = User::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id');
    }

    public function columns(): array
    {
        return [
            Column::make("Name", "name")
                ->sortable(),
            Column::make("Email", "email")
                ->sortable(),
            Column::make("Is active", "is_active")
                ->sortable(),
            Column::make("Hire date", "hire_date")
                ->sortable(),
            Column::make("Position", "position")
                ->sortable(),
            Column::make("Created at", "created_at")
                ->sortable(),
            Column::make("Updated at", "updated_at")
                ->sortable(),
        ];
    }
}
`

**Comment by artem-sainar** [comment]
I have the same problem. The table is drawn for one second and then lost with many console errors.

![image](https://github.com/user-attachments/assets/1591bba8-42a6-4e20-9785-3e1db377cd1a)

almost empty project with starter laravel kit ([Breeze and Livewire](https://laravel.com/docs/11.x/starter-kits#breeze-and-livewire))

**Reply by artem-sainar** [reply]
@ltmstudio If you used laravel kit, I found solution here https://livewire.laravel.com/docs/troubleshooting. It works for me.

try remove from your resources/js/app.js

![image](https://github.com/user-attachments/assets/c15b3a9c-c37c-43ee-bbdb-514e3a8ff978)

**Reply by ltmstudio** [reply]
Unfortunately, this option didn't work
![er](https://github.com/user-attachments/assets/4d974c1c-cf23-428e-bc9d-2113606a9cf2)

**Reply by lrljoe** [reply]
> @ltmstudio If you used laravel kit, I found solution here https://livewire.laravel.com/docs/troubleshooting. It works for me.
> 
> try remove from your resources/js/app.js
> 
> ![image](https://github.com/user-attachments/assets/c15b3a9c-c37c-43ee-bbdb-514e3a8ff978)
>  

Please advise as to whether you are injecting or bundling 
Livewire Core
Livewire Tables
